flatten_json.py

```diff
@@ -104,17 +104,19 @@ def unflatten_list(flat_dict, separator='_'):
 
     def _convert_dict_to_list(object_, parent_object, parent_object_key):
         if isinstance(object_, dict):
+            keys = []
```

Comment:
Can move it to the `except` section. Otherwise len(keys) on #112 will fail, when exception is raised from the conversion.